apps/rehabstod/src/schemas/userSchema.ts

```diff
@@ -2,6 +2,15 @@ import { z } from 'zod'
 import { vardenhetSchema } from './vardenhetSchema'
 import { vardgivareSchema } from './vardgivareSchema'
 
+export const DAYS_BETWEEN_SICK_LEAVES = {
+  MIN: 0,
```

Comment:
There's  a bug when I try to enter 0 it resets the value to the last saved value for both inputs. I can see here that the min limit is correct. Maybe it reads 0 as empty string and therefore resets it?

Reply:
Good catch, changed `convertValue` to look for `!Number.isNaN(val)` instead of `Boolean(val)` solved the issue.